lib/Drawing/Color.vala

```diff
@@ -441,16 +461,38 @@ namespace Granite.Drawing {
          * {@inheritDoc}
          */
         public void settings_deserialize (string s) {
-        
+
             var parts = s.split (";;");
-            
+
             R = double.min (uint8.MAX, double.max (0, int.parse(parts [0]))) / uint8.MAX;
             G = double.min (uint8.MAX, double.max (0, int.parse(parts [1]))) / uint8.MAX;
             B = double.min (uint8.MAX, double.max (0, int.parse(parts [2]))) / uint8.MAX;
             A = double.min (uint8.MAX, double.max (0, int.parse(parts [3]))) / uint8.MAX;
         }
-        
+
+        /**
+         * Returns a textual specification of this in the form `rgb (r, g, b)` or `rgba (r, g, b, a)`,
+         * where “r”, “g”, “b” and “a” represent the red, green, blue and alpha values respectively.
+         *
+         * r, g, and b are represented as integers in the range 0 to 255, and a is represented as
+         * floating point value in the range 0 to 1.
+         *
+         * Note that this string representation may lose some precision, since r, g and b are represented
+         * as 8-bit integers. If this is a concern, you should use a different representation.
+         *
+         * @return the text string
+         */
+        public string to_string () {
```

Comment:
Please mention the `Gdk.RGBA` struct here, along with the link to the `Gdk.RGBA.to_string` method so that people know how this is converted to string.